lib/lobbyist/v2/base.rb

```diff
@@ -35,6 +35,12 @@ def self.create_collection_from_response_with_model_name(response)
             list << self.new(element)
           end
           return list
+        elsif response[response.keys.first.to_s].is_a?(Array)
```

Comment:
Why have you added this functionality to this method? Shouldn't it belong in the create_collection_from_response similar to the logic in create_from_response ? Or do you need to modify the api to return the data to be used with the logic in create_collection_from_response?